DeepLabCut/deeplabcut_main.py

```diff
@@ -0,0 +1,180 @@
+import os
+import sys
+import numpy as np
+import pandas as pd
+import h5py
+import yaml
+import cv2
+import glob
+
+sys.path.append("..")
+import utilities as utils
+import definitions as defs
+import DeepLabCut.deeplabcut_parameters as dlc_params
+import DeepLabCut.deeplabcut_definitions as dlc_defs
+
+import deeplabcut
+
+from multiprocessing import freeze_support
+freeze_support()
+
+def main(deeplabcut_params: dlc_params.DeepLabCutParameters):
+
+    """
+    DeepLabCut algorithm
+    """
+    # Read danse parameters 
+    filepath: str = deeplabcut_params.paths[defs.Parameters.Path.FILEPATH]
+    datapath: str = deeplabcut_params.paths[defs.Parameters.Path.H5PATH]
+
+    project_folder: str    = deeplabcut_params.params[dlc_defs.Parameters.danse.PROJECT_FOLDER]
+    bodypart_names: list   = deeplabcut_params.params[dlc_defs.Parameters.danse.BODY_PART_NAMES].split(', ')
+    extracted_frames: list = deeplabcut_params.params[dlc_defs.Parameters.danse.EXTRACTED_FRAMES]
+    
+    extracted_frames = [int(frame) for frame in extracted_frames]
```

Comment:
Do we still have to do it, or danse passes this parameter in correct format now?